gulpfile.js

```diff
@@ -176,13 +193,14 @@ gulp.task('readme', () => {
 gulp.task('watch', (done) => {
   serverServe(done);
 
-  gulp.watch(`${config.src.scssPath}/**/*.scss`, gulp.series('css', serverReload));
-  gulp.watch(`${config.src.jsPath}/**/*.js`, gulp.series('js', serverReload));
+  // gulp.watch(`${config.src.scssPath}/**/*.scss`, gulp.series('css', serverReload));
```

Comment:
Should these be commented out, or just removed?

Reply:
I've left them commented in case we add CSS or other JS to the project in the future, since CSS and JS build tasks are still in the gulpfile.  I can remove them if y'all have a preference.